react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -37,8 +37,8 @@ class App extends Component {
 
   subtractItem = itemId => {
     const { bookSelected } = this.props;
-    const bookS = bookSelected.find(book => book.id === itemId);
-    if (bookS.quantity > 1) {
+    const { quantity } = bookSelected.find(book => book.id === itemId) || {};
+    if (quantity && quantity > 1) {
       this.props.subtractItem(itemId);
```

Comment:
in the top you have `const { bookSelected } = this.props;` destructure `substractItem` there, but this is just a comment for you to know that!